modules/thunder_article/thunder_article.post_update.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Thunder.
+ */
+
+/**
+ * Move scheduler view tab to local task on content list.
+ */
+function thunder_article_post_update_move_scheduler_to_content_list() {
```

Comment:
Should we be using the checklist thing for this update?

Reply:
I also asked @dbosen about that and we think it's fine to leave it out here. We are just importing a simple config and not modifying an existing one. 

@alexpott Could you think about a case, where this might fail?

Reply:
I thought the point of the checklist was not only to tell people about things that might fail but also to tell people what happened. If it is only about possible fails I agree - no need.

Reply:
Yes, for the previous pull request it was not so important. But now we also replace  the locked_content view. This should be communicated in the checklist thing.
And also I am not as light headed as I was yesterday ;)